dask/dataframe/rolling.py

```diff
@@ -0,0 +1,47 @@
+from functools import partial, wraps
+from itertools import count
+
+from toolz import merge
+import pandas as pd
+
+tokens = ('rolling-%d' % i for i in count(1))
+
+
+def rolling_chunk(func, part1, part2, window, *args):
+    if part1.shape[0] < window:
+        raise NotImplementedError("Window larger than partition size")
+    extra = window - 1
+    combined = pd.concat([part1[-extra:], part2])
+    applied = func(combined, window, *args)
+    return applied[extra:]
+
+
+def wrap_rolling(func):
+    """Create a chunked version of a pandas.rolling_* function"""
+    @wraps(func)
+    def rolling(arg, window, *args, **kwargs):
+        old_name = arg._name
+        new_name = next(tokens)
+        f = partial(func, **kwargs)
```

Comment:
That's what I had original, but that failed because `rolling_apply` takes a function, which resulted in an args tuple starting with a function. Could certainly make that a list though, idk.